guide/popular-topics/common-questions.md

```diff
@@ -98,6 +98,17 @@ user.send('<content>');
 
 <p class="tip">If you want to DM the user who sent the message, you can use `<message>.author.send()`.</p>
 
+### How do I tag a certain user in a message?
+
+```js
+const user = <client>.users.get('<id>');
+<message>.channel.send(`Hi, ${user}.`);
```

Comment:
Suggest to use <@id> as well.

Reply:
I'll change the first example to a `message.mentions.users.first()` snippet then, so that it's not as repetitive.